### ISSUE TITLE
Using with Nominet EPP

### ISSUE THREAD

**Issue by sembeeuk** [getpinga/fossbilling-epp-rfc#8]
I am trying to test this module with our Nominet account, initially with their OT&E sandbox. 

https://registrars.nominet.uk/uk-namespace/registration-and-domain-management/registration-systems/epp/

Nominet don't require the use of client side certificates, so I took your advice from another question and generated our own to satisfy the requirements. 

However I cannot get the pathing settings correct. 

The error is included below, the files are currently in the location shown in the error and that is how it is configured within the module, but is obviously incorrect. 

```
PHP Warning:  stream_socket_client(): Unable to set private key file `/var/www/fossbilling/key.pem' in /var/www/fossbilling/library/Registrar/Adapter/Nominet.php on line 1316
PHP Warning:  stream_socket_client(): Failed to enable crypto in /var/www/fossbilling/library/Registrar/Adapter/Nominet.php on line 1316
PHP Warning:  stream_socket_client(): Unable to connect to tlsv1.2://ote-epp.nominet.org.uk:700 (Unknown error) in /var/www/fossbilling/library/Registrar/Adapter/Nominet.php on line 1316
 Cannot connect to server 'ote-epp.nominet.org.uk':  0
```

Could you tell me what the path settings should be?
Many thanks.

**Comment by getpinga** [comment]
Please try putting it in the same path where is the module - /var/www/fossbilling/library/Registrar/Adapter , then set the same owner of the files as the module and try again. Let me know if does not work.

**Comment by sembeeuk** [comment]
The problem has been fixed with your assistance. 

I think the actual bit I was missing was the changing the owner, as just moving the files to that path on its own still returned the same error. Once I changed the owner, I then started getting other errors, which showed me that I needed to put my TAG in to all upper case to actually make the connection successfully. 

Testing the registration process then allowed me to confirm the domain was available, and move through to Checkout. when the test failed. However I don't think that is an issue with this module, so I will close it off.
